py3status/modules/netdata.py

```diff
@@ -59,12 +59,22 @@ class Py3status:
     low_traffic = 400
     med_speed = 60
     med_traffic = 700
-    nic = 'eth0'
+    nic = None
 
     def __init__(self):
         self.old_transmitted = 0
         self.old_received = 0
 
+    def post_config_hook(self):
+        if self.nic is None:
+            """ Get default gateway directly from /proc."""
+            with open("/proc/net/route") as fh:
+                for line in fh:
+                    fields = line.strip().split()
+                    if fields[1] != '00000000' or not int(fields[3], 16) & 2:
+                        continue
+                    self.nic = fields[0]
```

Comment:
really minor but I'd do

```
for ...:
    if condition:
        thing
        break
```

that way we don't do more work than needed.  Admittedly here we are never going to have much work.

I'm assuming that we can only ever have one gateway, is that the case.  The docstring should probably say how we auto select.

anyhow we can end up with `self.nic == None` (no wifi/ethernet etc) after this so we need to protect against that.